Makefile

```diff
@@ -5,6 +5,12 @@ HOSTNAME ?= $(shell hostname)
 # HOSTNAME is the name of the physical host
 export HOSTNAME := $(HOSTNAME)
 
+# Exposed ports to the host (default to same as in container)
+export U7S_ETCD_EXPOSED_PORT ?= 2379
```

Comment:
Probably these envs should rather have the common prefix (U7S_EXPOSED_PORT_ or perhaps just U7S_PORT_)